hummingbot/strategy/cross_exchange_market_making/cross_exchange_market_making.pyx

```diff
@@ -64,29 +66,32 @@ cdef class CrossExchangeMarketMakingStrategy(StrategyBase):
     def __init__(self,
                  market_pairs: List[CrossExchangeMarketPair],
                  min_profitability: float,
-                 trade_size_override: Optional[float] = 0.0,
+                 order_amount: Optional[float] = 0.0,
                  order_size_taker_volume_factor: float = 0.25,
                  order_size_taker_balance_factor: float = 0.995,
                  order_size_portfolio_ratio_limit: float = 0.1667,
                  limit_order_min_expiration: float = 130.0,
-                 cancel_order_threshold: float = -1,
-                 active_order_canceling: bint = True,
+                 adjust_order_enabled: bool = True,
```

Comment:
There's no documentation for what the `adjust_order_enabled` flag does, and it's still not clear to me after reviewing the code.

Reply:
Refer to carlo's comments. This is the feature which he requested, that does not change based on the top bid / ask. I will add the doc.